spec/swagger.yaml

```diff
@@ -138,23 +121,34 @@ paths:
               caffeine:
                 type: integer
                 description: mg of caffeine per 100 ml/mg/unit
+              alcohol:
+                type: integer
+                description: vol % alcohol without decimal seperator
+                example: 450
               price:
                 type: integer
                 default: 150
+              barcode:
+                type: string
+              image:
```

Comment:
How is this supposed to work?
Currently (mete/v1) we are `POST`ing (or `PATCH`ing) a file called `logo` along with the other parameters.
(Which isn't documented, yet.)